### DISCUSSION TITLE
时间轴损坏的文件播放时无法显示完整的进度条

### DISCUSSION THREAD

**Discussion by AcerolaShinobu** [hooke007/mpv_PlayKit#430] [mpv 通用]
![ezgif-4-8cf0ceefa2](https://github.com/hooke007/MPV_lazy/assets/38858821/98edb728-b20a-4136-a1c9-d3e00dafd616)
MPV播放时随播放进度增加最大进度条。
![图片](https://github.com/hooke007/MPV_lazy/assets/38858821/2aa406fa-f69b-450b-8d62-a6c6c991eab5)
MPC播放器可以正常显示完整的进度条。
应该是下载的视频文件存在某种问题
有没有办法使MPV像MPC一样播放这类文件时显示完整进度条呢。
视频文件:https://github.com/hooke007/MPV_lazy/discussions/430#discussioncomment-8816253

**Comment by AcerolaShinobu** [comment]
https://www.youtube.com/watch?v=VNdHd1asf9s
![图片](https://github.com/hooke007/MPV_lazy/assets/38858821/79ce4592-bf50-4f14-9787-fc7de5ea66e6)
用MPV在线播放倒是没有问题

**Comment by hooke007** [comment]
文件不大直接传github 你的地址没有访问权限

**Reply by hooke007** [reply]
另外这个文件我下下来看是正常的，你的下载下崩了。
![MPV-00:00:14 139-N0001](https://github.com/hooke007/MPV_lazy/assets/41094733/e08ad9b3-f839-48c5-af95-28b8b21ced66)

**Reply by AcerolaShinobu** [reply]
是的，应该是用的下载器有问题，下载的视频都这样，因为我之前一直都用MPC播放器，所以没发现

**Comment by hooke007** [comment]
不管是ffprobe，mediainfo还是其它播放器都是没时长。
<img width="400" alt="截屏2024-03-17 13 46 18" src="https://github.com/hooke007/MPV_lazy/assets/41094733/40e8571c-99a5-4a25-bbb2-a0e53cd0c886"> <img width="400" alt="截屏2024-03-17 13 50 38" src="https://github.com/hooke007/MPV_lazy/assets/41094733/99ce6483-a16b-4055-b1af-2fafb2bf7599">

mpc可能用了其它探测方法，除非报告给mpv，但是我觉得针对这类损坏的文件可能没有特意实现此功能的价值

**Reply by AcerolaShinobu** [reply]
那我还是重新下载吧。。

**Comment by hooke007** [comment]
PS.正确的传文件方式是直接传进对话里而不是开仓库。。。

[you can be king again.webm](https://github.com/hooke007/MPV_lazy/assets/41094733/b8ab7944-d791-4d27-9f40-6bb6a366ae8b)

**Reply by AcerolaShinobu** [reply]
好的，我这边显示"This video is too big.Try again with a file size less than 10MB."，所以扔仓库里了

**Reply by hooke007** [reply]
哦可能是账户等级限制，，，
